doc/source/how-to/setting-up.rst

```diff
@@ -0,0 +1,528 @@
+Setting Up Your Development Environment
+=======================================
+Before you can contribute to any ``PyAnsys`` project, you must set up
+your developer environment.
+
+Python
+------
+.. raw:: html
+    
+    <div align="center">
+      <img src="https://github.com/python/pythondotorg/raw/main/static/community_logos/python-logo-inkscape.svg">
+    </div>
+
+All ``PyAnsys`` projects require a Python interpreter for interacting
+with the software. Therefore, you must ensure that at least one Python
+interpreter is installed on your local machine.
+
+Installation
+~~~~~~~~~~~~
+There are multiple ways to install a Python package on your local machine:
+
+- Use an official installer from the `official Python download section <https://www.python.org/downloads/>`_.
+- Install via a package manager or "store" on your machine.
+
+.. warning:: 
+
+  Ensure that you install Python from an official channel. Do not trust
+   third-party websites or download executable content from them.
+
+.. tabs::
+
+    .. group-tab:: Windows
+
+        To install Python on a machine running Windows:
+        
+        1. Download the `latest stable Python version for Windows <https://www.python.org/downloads/windows/>`_.
+        2. Execute the installer, referring to  `Using Python on
+           Windows <https://docs.python.org/3/using/windows.html>`_ for
+           detailed installation information.
+
+    .. group-tab:: macOS
+
+        To install Python on a machine running the macOS:
+        
+        1. Download the `latest stable Python version for macOS <https://www.python.org/downloads/macos/>`_.
+        2. Execute the installer, referring to `Using Python on
+           a Mac <https://docs.python.org/3/using/mac.html>`_ for
+           detailed installation information.
+
+        .. note::
+
+            It is likely that your macOS distribution already comes with some
+            version of Python installed. For more information, see :ref:`Verification`.
+
+    .. group-tab:: Linux/UNIX
+
+        To install Python on a machine running Linux/UNIX:
+        
+        1. Download the `latest stable Python version for Linux/UNIX <https://www.python.org/downloads/source/>`_.
+        2. Decompress the source code and follow the installation instructions in the
+           ``README.rst`` file. For more information, see `Using Python on Unix Platforms
+           <https://docs.python.org/3/using/unix.html>`_.
+
+        .. note::
+
+            It is likely that your Linux/UNIX distribution already comes with some
+            version of Python installed. For more information, see :ref:`Verification`.
+
+
+Verification
+~~~~~~~~~~~~
+Once your Python installation is complete, verify it with:
+
+.. tabs::
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            py --version
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            py --version
+
```

Comment:
Why not directly group them under "Windows", instead of having two separate ones? That way your tab will also move, when you click around for Windows in the previous group tab :)

Reply:
If the commands differ (as I have seen in other code blocks) you could differentiate them with code comments such as:
```bash
# For CMD
...

# For Powershell
...
```

Or even add a second level of tabs for Windows. See https://github.com/ansys/api-guidance/blob/main/doc/index.rst for an example.

Reply:
I used this:

```
.. tabs::

    .. code-tab:: pwsh PowerShell

       C:\venv\pymapdl\Scripts\Activate.ps1

    .. code-tab:: doscon Command prompt (cmd)

       C:\venv\pymapdl\Scripts\activate.bat
```

Reply:
Fixed in 181bfcf.